schedule/microos/suse_microos_dvd.yaml

```diff
@@ -2,7 +2,15 @@ name:           suse_microos
 description:    >
     Maintainer: jalausuch@suse.com, qa-c@suse.de.
     SUSE MicroOS tests
+conditional_schedule:
+    bootloader:
+        ARCH:
+            'aarch64':
+                - installation/bootloader_uefi
+            'x86_64':
+                - installation/bootloader
 schedule:
+    - '{{bootloader}}'
     - installation/bootloader
```

Comment:
fully agree with Martin here and in any case I assume you need replace this line and not add new one . current version would call `installation/bootloader` two times for 'x86_64'